pennylane/data/dataset.py

```diff
@@ -18,13 +18,25 @@
 from abc import ABC
 from glob import glob
 import os
-import dill
-import zstd
 
 from pennylane import Hamiltonian
 from pennylane.pauli import string_to_pauli_word
 
 
+def _import_zstd_dill():
+    """Import zstd and dill."""
+    try:
+        # pylint: disable=import-outside-toplevel, unused-import, multiple-imports
+        import zstd, dill
+    except ImportError as Error:
+        raise ImportError(
+            "This feature requires zstd and dill."
+            "They can be installed with: pip install zstd dill."
```

Comment:
```suggestion
            "This feature requires zstd and dill."
            "They can be installed with:\n\n pip install zstd dill."
```
Minor, but having the command on a new line might help users identify the command